src/build_helper/lib.rs

```diff
@@ -40,7 +40,9 @@ pub fn gnu_target(target: &str) -> String {
 }
 
 pub fn cc2ar(cc: &Path, target: &str) -> PathBuf {
-    if target.contains("musl") || target.contains("msvc") {
+    if target.contains("msvc") {
+        PathBuf::from("lib")
```

Comment:
It fixes the sanity check. In all cases where `cc2ar` is called or the `.ar()` of the `Build` is called, the result is never used on msvc, except for the sanity check. But I'll move it to a separate PR and do the `Option` thing.